src/draw/lv_draw_private.h

```diff
@@ -111,6 +111,13 @@ struct lv_draw_unit_t {
      */
     int32_t (*evaluate_cb)(lv_draw_unit_t * draw_unit, lv_draw_task_t * task);
 
+    /**
+     *
+     * @param draw_unit
+     * @return
+     */
+    int32_t (*wait_for_finish_cb)(lv_draw_unit_t * draw_unit);
```

Comment:
@FASTSHIFT I believe it might be useful for you too

Reply:
```
LVGL thread					VGLite thread				GPU HW
task1						queued					IDLE
						vg_lite_flush()				task1 is flushed to GPU
task2						queued					BUSY (with task1)

task3						queued					BUSY (still with task1)
				
task4						qeueud					IDLE (task1 ready now)
						vg_lite_flush()				task2, task3, task4 are flushed to GPU
						task1->ready = true
no more tasks in the queue
finish_cb()					vg_lite_finish()			GPU have to complete all tasks previously flushed
						task2->ready = true
						task3->ready = true
						task4->ready = true
```

Reply:
I will rename the `wait_for_finish_cb` to just `finish_cb' because there is no waiting at all at the LVGL thread, it will be just a signal sent to VGLite thread to finish all pending tasks and send the ready status back.